java/ingest/ingest-batcher-job-creator/src/test/java/sleeper/ingest/batcher/job/creator/IngestBatcherJobCreatorLambdaIT.java

```diff
@@ -59,13 +59,13 @@ public class IngestBatcherJobCreatorLambdaIT extends LocalStackTestBase {
     @BeforeEach
     void setUp() {
         createBucket(instanceProperties.get(CONFIG_BUCKET));
-        DynamoDBTableIndexCreator.create(dynamoClient, instanceProperties);
-        DynamoDBIngestBatcherStoreCreator.create(instanceProperties, dynamoClient);
+        DynamoDBTableIndexCreator.create(dynamoClientV2, instanceProperties);
+        DynamoDBIngestBatcherStoreCreator.create(instanceProperties, dynamoClientV2);
         instanceProperties.set(DEFAULT_INGEST_BATCHER_INGEST_QUEUE, STANDARD_INGEST.toString());
         instanceProperties.set(DEFAULT_INGEST_BATCHER_MIN_JOB_SIZE, "0");
         instanceProperties.set(INGEST_JOB_QUEUE_URL, sqsClient.createQueue(UUID.randomUUID().toString()).getQueueUrl());
```

Comment:
This is still using SQS SDK v1 to create the queue. If you use the helper on LocalStackTestBase that can be migrated separately.